osu.Game/Graphics/Cursor/OsuTooltipContainer.cs

```diff
@@ -70,6 +71,7 @@ public OsuTooltip()
                         TextSize = text_size,
                         Padding = new MarginPadding(5),
                         Font = @"Exo2.0-Regular",
+                        Text = " ",
```

Comment:
what is this?

Reply:
Text needs to be changed so that the first tooltip wont show if text is empty. Basically

Reply:
this shouldn't be necessary / can't exist